src/Exception/TransformerExceptionInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Dhii\Transformer\Exception;
+
+use Dhii\Exception\ThrowableInterface;
+use Dhii\Transformer\TransformerInterface;
+
+/**
+ * An exception thrown in relation to a transformer.
+ *
+ * @since [*next-version*]
+ */
+interface TransformerExceptionInterface extends ThrowableInterface
+{
+    /**
+     * Retrieves the transformer instance that erred, if any.
+     *
+     * @since [*next-version*]
+     *
+     * @return TransformerInterface|null
+     */
+    public function getTransformer();

```

Comment:
I think this should be in the empty `TransformerAwareInterface` and this interface extends it.